packages/charts/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -1121,18 +1136,15 @@ export class VerticalStackedBarChartBase
    * 1. selection: if the user clicks on it
    * 2. hovering: if there is no selected legend and the user hovers over it
    */
-  private _legendHighlighted = (legendTitle: string) => {
-    return (
-      this.state.selectedLegend === legendTitle ||
-      (this.state.selectedLegend === '' && this.state.activeLegend === legendTitle)
-    );
+  private _legendHighlighted = (legendTitle: string): boolean => {
```

Comment:
>_legendHighlighted [](http://example.com/codeflow?start=10&length=18)

_isLegendHighlighted

Reply:
_legendHighlighted was already there, so didn't change the name. Will change

Reply:
updated